### ISSUE TITLE
Repository set to https://wwwhub.com

### ISSUE THREAD

**Issue by peteruithoven** [davglass/license-checker#117]
Some packages get `https://wwwhub.com/...` as `repository`, 

For example [@doodle3d/clipper-js](https://www.npmjs.com/package/@doodle3d/clipper-js) ([repo](https://github.com/Doodle3D/clipper-js)): 
```
├─ @doodle3d/clipper-js@1.0.3
│  ├─ licenses: MIT
│  ├─ repository: https://wwwhub.com/Doodle3D/clipper-js
│  ├─ publisher: Casper @ Doodle3D
│  └─ licenseFile: /home/peteruithoven/Projects/Doodle3D/dev/Doodle3D-App/node_modules/@doodle3d/clipper-js/LICENSE
```

Also seen in: https://github.com/davglass/license-checker/issues/99#issue-230505405

**Comment by davglass** [comment]
Fixed, pushed in `license-checker@13.0.3`

**Comment by peteruithoven** [comment]
Thanks @davglass !
